conftest.py

```diff
@@ -5,15 +5,45 @@
 the proper patching.
 """
 
+import random
 from functools import partial
 from pathlib import Path
 from urllib.parse import urljoin
 
 import pytest
+from django.apps import apps
 from django.contrib.auth import get_user_model
+from django.db import connection
 from django.test import override_settings
 
 
+@pytest.fixture(scope="session")
+def django_db_setup(django_db_setup, django_db_blocker):
+    """Force db sequences to use random numbers."""
+    with django_db_blocker.unblock():
+        randomize_db_sequences()
+
+
+def randomize_db_sequences():
+    """Randomize db sequences."""
+    # shamelessly inspired on https://pytest-django.readthedocs.io/en/latest/database.html#randomize-database-sequences
```

Comment:
I don't have any qualm on which one is better, but this popped up in my feed recently: https://adamj.eu/tech/2023/07/04/django-test-random-order-querysets/